dev-python/javasphinx/javasphinx-0.9.15.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} pypy{,3} )
+
+inherit distutils-r1
+
+DESCRIPTION="Sphinx extension for documenting Java projects"
+HOMEPAGE="https://github.com/bronto/javasphinx"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~x64-solaris"
+
+RDEPEND="
+	>=dev-python/javalang-0.10.1
+	dev-python/lxml
```

Comment:
All the Python dependencies should have the same Python version USE flags.  cf. https://wiki.gentoo.org/wiki/Project:Python/distutils-r1#PYTHON_USEDEP

The same applies to all others.

Reply:
I'd like to make some more thorough tests on this, as I seem to remove it after it led to some failures. But not sure why. Will test it and sort it out.

Reply:
Tested and fixed it as well in the affected ebuilds.